tasks/57_pig-latin/pig-latin.js

```diff
@@ -0,0 +1,25 @@
+
+
+export function translatePigLatin(strInput) {
+  let ret = strInput.split('');
```

Comment:
What is `ret`? ret as in return? Why do we need to know that in advance? Make sure to always use domain-specific names. What IS this variable? What does it CONTAIN?